ZenPacks/zenoss/LinuxMonitor/tests/plugindata/rpm

```diff
@@ -0,0 +1,29 @@
+export PATH=$PATH:/bin:/sbin:/usr/bin:/usr/sbin; `which rpm || echo true` -qa --qf "%{NAME}__FIELD__%{VERSION}__FIELD__%{RELEASE}__FIELD__%{INSTALLTIME}__FIELD__%{VENDOR}__FIELD__%{SUMMARY}\n"; echo "_SPLIT_"; if command -v dpkg dpkg >/dev/null 2>&1; then for file_list in `ls /var/lib/dpkg/info/*.list`; do stat_result=$(stat --format=%Y "$file_list"); printf "_field:%s_field:%s\n" "$(basename $file_list .list)" "$stat_result"; done; dpkg-query -W -f='_field:${Package}_field:${Version}_field:${Maintainer}_field:${Description}\n'; fi
```

Comment:
Can you format this line with linebreaks so that its easier to understand.?
Also Prefer $() to backtics here.